tdrs-frontend/src/components/PrivateRoute/PrivateRoute.js

```diff
@@ -33,12 +40,22 @@ function PrivateRoute({ children, title }) {
     }
   }, [authenticated, authLoading, dispatch, navigate])
 
-  return authenticated ? (
-    <PrivateTemplate title={title}>
-      {children}
-      <IdleTimer />
-    </PrivateTemplate>
-  ) : null
+  if (authenticated) {
+    return (
+      <PermissionGuard
+        requiresApproval={requiresApproval}
+        requiredPermissions={requiredPermissions}
+        notAllowedComponent={<Navigate to="/home" />}
```

Comment:
@reitermb a11y question for ya - here we redirect an unauthorized user back to the home page if they try to access a blocked route. @ADPennington brought up in tabletop that redirecting without warning might be an issue. any thoughts? fwiw, this same logic is used in the `Profile.jsx` component.

Reply:
This scenario would involve an attempt to manually navigate to a page you (a user) hasn't been granted access to, right? i.e. typing the route into the URL directly?

If we're dropping them on some kind of intermediary page (like a 404) before redirecting them after some time interval then that would bring up some a11y concerns related to making sure people have [enough time](https://www.w3.org/WAI/WCAG22/Understanding/enough-time#:~:text=This%20guideline%20focuses%20on%20ensuring,them%20to%20complete%20their%20tasks.) to read and interact with a given page—but I think automatically redirecting straight back to home is okay given that our navigation design is really aimed here at being about keeping users out of pages they're not allowed access to rather than giving them insight into the routes of those pages.

cc @ADPennington